### ISSUE TITLE
Py3.10 / macOS Feast wheel missing "protos"

### ISSUE THREAD

**Issue by mikeyk** [feast-dev/feast#2639]
## Expected Behavior 
`pip install feast`
`feast init` (or anything) works

## Current Behavior

Any feast command fails with:
```
    from feast.protos.feast.types.Value_pb2 import (
ModuleNotFoundError: No module named 'feast.protos'
```

## Steps to reproduce
1. Install Python 3.10 / x64_64 version of the macOS wheels for anything above Feast `0.20.0`
2. Run any feast command

Versions `0.19` and earlier work fine, and it looks like the `protos` are available for Python 3.9, just not 3.10

### Specifications

- Version: 0.20.x
- Platform: macOS
- Subsystem:

**Comment by achals** [comment]
Thanks @mikeyk  - we've landed some changes in our packaging that should likely fix this and a few other issues (#2630). we're planning on cutting a new version with these fixes and an improved release pipeline to catch these kinds of issues sooner.

**Comment by achals** [comment]
BTW @mikeyk was this an M1 mac x86_64?

**Comment by mikeyk** [comment]
> BTW @mikeyk was this an M1 mac x86_64?

That's right -- it's a Rosetta-based x86_64 virtualenv (so Python 3.10 installed via an x86_64 homebrew environment then with a virtualenv created from it)

**Comment by pyalex** [comment]
Fixed #2687
